api/internal/utils/pid.go

```diff
@@ -25,9 +25,13 @@ import (
 )
 
 // WritePID write pid to the given file path.
-func WritePID(filepath string) error {
-	if _, err := os.Stat(filepath); err == nil {
-		return fmt.Errorf("instance of Manager API already running: a pid file exists in %s", filepath)
+func WritePID(filepath string, forceStart bool) error {
+	if pid, err := ReadPID(filepath); err == nil {
+		if !forceStart {
+			return fmt.Errorf("instance of Manager API already running: a pid file exists in %s", filepath)
+		} else {
+			fmt.Printf("force starting new instance. another instance of Manager API is already running with pid %d\n", pid)
```

Comment:
`is already` => `maybe` 
Is this would be better? Since the old manager API maybe is not running.

Reply:
sorry for the late reply. Yeah kinda makes sense though. If due to some reason the pid file cleanup didn't take place which is obviously very unlikely. Yet `maybe` sounds good.